examples/mongoose.test.js

```diff
@@ -2045,4 +2045,54 @@ describe('Mongoose', function() {
     assert.equal(profilingLevel, 'off');
     // acquit:ignore:end
   });
+  describe('mongoose-find-async', function() {
+    it('find', async function() {
+        // acquit:ignore:start
+        await mongoose.disconnect();
+        // Connect to a MongoDB server running on 'localhost:27017' and use the
+        // 'test' database.
+        await mongoose.connect('mongodb://localhost:27017/test', {
+          useNewUrlParser: true // Boilerplate for Mongoose 5.x
+        });
+        const Character = mongoose.model('Character', mongoose.Schema({
+          name: String,
+          age: Number,
+          rank: String
+        }));
+        // acquit:ignore:end
+        await Character.create([
+          { name: 'Jean-Luc Picard', age: 59, rank: 'Captain' },
+          { name: 'William Riker', age: 29, rank: 'Commander' },
+          { name: 'Deanna Troi', age: 28, rank: 'Lieutenant Commander' },
+          { name: 'Geordi La Forge', age: 29, rank: 'Lieutenant' },
+          { name: 'Worf', age: 24, rank: 'Lieutenant' }
+        ]);
+        // The query to find all the Lieutenants
+        const query = await Character.find({rank: 'Lieutenant'}); // will return Worf and Forge
```

Comment:
Minor nitpicks: space between `{` and `rank`, and between `nant'` and `}`. Also, "La Forge" :)